AC3.2-MidUnit-6-Exam/DoubleVideoViewController.swift

```diff
@@ -7,25 +7,74 @@
 //
 
 import UIKit
+import AVKit
+import AVFoundation
+import MobileCoreServices
 
-class DoubleVideoViewController: UIViewController, CellTitled {
+class DoubleVideoViewController: UIViewController, CellTitled, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
+    
+    var myURL: URL?
+    var avplayer: AVPlayer!
+    var userPlayRate: Float = 1.0
+    var userPlaying: Bool = true
     var titleForCell: String = "Double Video"
-
+    
+    @IBOutlet var doubleVideoOverLay: UIView!
     @IBOutlet weak var videoContainerTop: UIView!
     @IBOutlet weak var videoContainerBottom: UIView!
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         // don't f with this line
         self.edgesForExtendedLayout = UIRectEdge(rawValue: 0)
     }
-
+    
     override func viewDidLayoutSubviews() {
         // update your layers' frames here
+        guard let sublayers = self.videoContainerTop.layer.sublayers else { return }
+        for layer in sublayers {
+            layer.frame = self.videoContainerTop.bounds
+        }
     }
     
     @IBAction func loadVideo(_ sender: UIButton) {
+        let imagePickerController = UIImagePickerController()
+        imagePickerController.sourceType = .photoLibrary
+        imagePickerController.mediaTypes = [String(kUTTypeMovie)]
```

Comment:
I like that you chose to limit to only movies. This wasn't a requirement but makes sense.